prime-router/src/main/kotlin/azure/HttpUtilities.kt

```diff
@@ -16,13 +17,6 @@ import java.time.OffsetDateTime
 import java.time.ZoneOffset
 import java.time.format.DateTimeFormatter
 
-enum class ReportStreamEnv(val urlPrefix: String) {
-    TEST("https://test.prime.cdc.gov"),
-    LOCAL("http://" + (System.getenv("PRIME_RS_API_ENDPOINT_HOST") ?: "localhost") + ":7071"),
-    STAGING("https://staging.prime.cdc.gov"),
-    PROD("not implemented"),
```

Comment:
Is this change in functionality intended?  Currently, if tests were run in prod they would have no effect because of this value. Now the test would use the actual PROD url. Do we want to be able to run tests against real prod endpoints?

Reply:
There was really no check in the code for "not implemented".  It would have just crashed due to not having a proper URL.  I changed it on purpose to allow for the tests, but if we do not want to test in prod a new check somewhere should be done against Environment.PROD.

Reply:
Wouldn't crashing be better than unintentionally running tests that could insert data into prod?

Reply:
This is more of a business rule question: would we want to run these tests in prod as a legitimate use case ever?  Looks like we have not wanted that in the past (since it would have crashed), not sure if we want that in the future. This change was the only functional change to the PR as far as I could tell.

Reply:
It was decided to not allow tests to run in prod, so the environment option for the CLI does not allow prod.  This is mainly because the ignore org does not exist in prod.